packer/ubuntu/ubuntu1604.json

```diff
@@ -0,0 +1,215 @@
+          "modifyvm",
+          "{{.Name}}",
+          "--memory",
+          "2048"
+        ],
+        [
+          "modifyvm",
+          "{{.Name}}",
+          "--cpus",
+          "4"
+        ]
+      ],
+      "virtualbox_version_file": ".vbox_version",
+      "vm_name": "ubuntu1604"
+    },
+    {
+      "name": "build",
+      "boot_command": [
+          "<enter><wait><f6><esc><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+          "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+          "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+          "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+          "/install/vmlinuz<wait>",
+          " auto<wait>",
+          " console-setup/ask_detect=false<wait>",
+          " console-setup/layoutcode=us<wait>",
+          " console-setup/modelcode=pc105<wait>",
+          " debconf/frontend=noninteractive<wait>",
+          " debian-installer=en_US<wait>",
+          " fb=false<wait>",
+          " initrd=/install/initrd.gz<wait>",
+          " locale=en_US<wait>",
+          " kbd-chooser/method=us<wait>",
+          " keyboard-configuration/layout=USA<wait>",
+          " keyboard-configuration/variant=USA<wait>",
+          " grub-installer/bootdev=/dev/sda<wait>",
+          " noapic<wait>",
+          " net.ifnames=0<wait>",
+          " hostname={{ user `hostname` }}<wait>",
+          " preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed-1604.cfg<wait>",
+          " -- <wait>",
+          "<enter><wait>"
+      ],
+      "disk_size": 65536,
+      "guest_additions_path": "VBoxGuestAdditions.iso",
+      "guest_os_type": "Ubuntu_64",
+      "hard_drive_interface": "sata",
+      "headless": "{{ user `headless` }}",
+      "http_directory": "http",
+      "iso_checksum": "{{ user `iso_checksum` }}",
+      "iso_checksum_type": "sha1",
+      "iso_urls": [
+        "{{ user `iso_path` }}/{{ user `iso_name` }}",
+        "{{ user `iso_url` }}"
+      ],
+      "shutdown_command": "echo '{{ user `ssh_password` }}'|sudo -S shutdown -P now",
+      "ssh_password": "{{ user `ssh_password` }}",
+      "ssh_username": "{{ user `ssh_username` }}",
+      "ssh_wait_timeout": "10000s",
+      "type": "virtualbox-iso",
+      "vboxmanage": [
+        [
+          "modifyvm",
+          "{{.Name}}",
+          "--memory",
+          "2048"
+        ],
+        [
+          "modifyvm",
+          "{{.Name}}",
+          "--cpus",
+          "4"
+        ]
+      ],
+      "virtualbox_version_file": ".vbox_version",
+      "vm_name": "ubuntu1604"
+    }
+  ],
+  "post-processors": [[
+    {
+      "keep_input_artifact": false,
+      "output": "box/ubuntu1604-netplugin.box",
+      "type": "vagrant"
+    },
+    {
+      "type": "atlas",
+      "token": "{{ user `atlas_token` }}",
+      "artifact": "contiv/ubuntu1604-netplugin",
```

Comment:
agree; then we can start testing with volplugin and not have it be
confusing. :D

On Tue, Jul 19, 2016 at 10:08 AM, Sukhesh Halemane <notifications@github.com
> wrote:

> In packer/ubuntu/ubuntu1604.json
> <https://github.com/contiv/build/pull/71#discussion_r71380873>:
>
> > +        ]
> > +      ],
> > +      "virtualbox_version_file": ".vbox_version",
> > +      "vm_name": "ubuntu1604"
> > +    }
> > +  ],
> > +  "post-processors": [[
> > +    {
> > +      "keep_input_artifact": false,
> > +      "output": "box/ubuntu1604-netplugin.box",
> > +      "type": "vagrant"
> > +    },
> > +    {
> > +      "type": "atlas",
> > +      "token": "{{ user `atlas_token` }}",
> > +      "artifact": "contiv/ubuntu1604-netplugin",
>
> that should be fine. previous box was named contiv/ubuntu1504-netplugin.
> So, i kept the same format.
> changing it to contiv/ubuntu1604 sounds better
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/contiv/build/pull/71/files/cc73aa4bcbe0698a9fb0a05fd744d822b298dc02#r71380873>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABJ67ySucB022GL2HqjLtiXOGyM-8Lbks5qXQSegaJpZM4JPQTt>
> .
>